simpleflow/storage.py

```
@@ -16,13 +16,26 @@ def get_connection(host):
     return connection.S3Connection(host=host)
 
 
+def sanitize_bucket_and_host(bucket):
+    """
+    if bucket is in following format : 'xxx.amazonaws.com/bucket_name',
+    Returns a 2-values tuple ('bucket_name', 'xxx.amazonaws.com')
+    """
+    if "/" in bucket:
+        host, bucket = bucket.split('/')
```

/ is not allowed in bucket, so we may want to raise a real error ?